vllm/sampling_params.py

```diff
@@ -70,6 +71,8 @@ class SamplingParams:
         stop_token_ids: List of tokens that stop the generation when they are
             generated. The returned output will contain the stop tokens unless
             the stop tokens are special tokens.
+        include_stop_in_output: Whether to include the stop strings in output
```

Comment:
`include_stop_str_in_output` looks a bit clearer, WDYT?

```suggestion
        include_stop_str_in_output: Whether to include the stop strings in output
```